website/content/partials/datasource/hcp-packer-image/Config-required.mdx

```diff
@@ -2,7 +2,16 @@
 
 - `bucket_name` (string) - The name of the bucket your image is in.
 
-- `iteration_id` (string) - The name of the iteration Id to use when retrieving your image
+- `channel` (string) - The name of the channel to use when retrieving your image.
+  Either this or `iteration_id` MUST be set.
+  Mutually exclusive with `iteration_id`.
+  If using several images from a single iteration, you may prefer
+  sourcing an iteration first, and referencing it for subsequent uses,
+  as getting the iteration induces a potentially billable request.
+
+- `iteration_id` (string) - The ID of the iteration to use when retrieving your image
+  Either this or `channel` MUST be set.
+  Mutually exclusive with `channel`
```

Comment:
kind of a nit but just to make this a bit clearer i think we could say something like this ```suggestion
- `channel` (string) - The name of the channel to use when retrieving your image.
  Either this or `iteration_id` MUST be set.
  Mutually exclusive with `iteration_id`.
  If using several images from a single iteration, you may prefer
  using an `hcp_packer_iteration` datasource first to source that iteration, and referencing it for subsequent uses,
  as each `hcp_packer_image` with the channel name set will generate a billable HCP Packer provision request

- `iteration_id` (string) - The ID of the iteration to use when retrieving your image
  Either this or `channel` MUST be set.
  Mutually exclusive with `channel`
```

Reply:
okay this one is borked trying again >_<